platform/source/lib/ax/platform/container.py

```diff
@@ -229,18 +235,25 @@ def add_volumes(self, volumes):
     def get_volume(self, name):
         return self.vmap.get(name, None)
 
-    def add_env(self, name, value=None, value_from=None):
+    def add_env(self, name, value=None, value_from=None, value_from_secret=None):
```

Comment:
Will ```value_from``` and ```value_from_secret``` be mutually exclusive? If so, can we simply use a single parameter to this function? Say ```value_from=ENV_FIELD_SELECTOR``` and ```value_from=ENV_SECRET_SELECTOR``` where ENV_FIELD_SELECTOR and ENV_SECRET_SELECTOR are globals that we define.